src/lib.rs

```diff
@@ -499,6 +499,15 @@ pub struct Config {
     max_send_udp_payload_size: usize,
 }
 
+// See https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#section-15
+fn is_reserved_version(version: u32) -> bool {
+    version == 0x00000000 ||
+        version
+            .to_le_bytes()
+            .iter()
+            .all(|b| (b & 0b0000_1111) == 0b0000_1010)
```

Comment:
I _think_ this whole thing could be simplified to just:
```rust
    version & 0xfafafafa == version
```
?

The `0xfafafafa` can then also be declared as a named const, so we don't hardcode it, e.g.
```rust
const RESERVED_VERSION_MASK: u32 = 0xfafafafa
```
or something like that.